extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -88,6 +88,12 @@ const LAUNCHABLE_COMPONENTS = {
   },
 };
 
+/** @const {!Array<string>} */
+const INTERACTIVE_ATTR_NAMES = [
+  '[interactive="true"]',
+  '[interactive=""]',
+  '[interactive]'];
```

Comment:
I find this much more complicated than what we had before, we replaced one line with multiple lines of configuration and loops and string manipulations :(
Also, `amp-twitter[interactive]:not([interactive=false])` might be better :)

Jon's call!

Reply:
Revised it. LMK what you think.